.zenodo.json

```diff
@@ -95,6 +95,9 @@
     {
       "name": "Duek, Or"
     },
+    {
+      "name": "Daniel, Jonathan"
```

Comment:
@jond01 I've added you to the Zenodo fragment, and you will be listed as an author in the [Zenodo archive](https://doi.org/10.5281/zenodo.591597). Please let me know if you would like your name to be rendered otherwise, or if you have an affiliation and/or [ORCID](https://orcid.org/) you would to use. I can also remove you if you'd rather not be listed as an author.

Reply:
Thanks a lot, what an honour. That's fine as is.
Looking forward to the new release!